### PR TITLE
UI component: MarketingTemplateBenefitBox

### PR DESCRIPTION
**Pull request by piotrkonowrocki**
# [UI component: MarketingTemplateBenefitBox](https://app.shortcut.com/oazo-apps/story/13980/ui-component-simple-slider-card)

image
  
## Changes 👷‍♀️

- Created `MarketingTemplateBenefitBox` component.
- Removed unused temporary icons.
- Changed the way `index` file of icons work - they have to be exported as a part of bigger interface as it's no possible to pass function from `getServerSideProps` to the next-js page.

## Additional notes 🗒️

- Icons are still incorrect; all slides are using the same icon and they placement is also off due to issues with svg files that will be addresses as separated story.

### CODE REVIEW

File: features/marketing-layouts/components/MarketingTemplateBenefitBox.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        {title}
+      </Heading>
+      {description && (
+        <Text as="p" variant="paragraph2" sx={{ color: 'neutral80' }}>
+          {description}
+        </Text>
+      )}
+      {list && (
+        <Box as="ul" sx={{ mt: '12px', pl: '24px' }}>
+          {list.map((item) => (
```
Comment (piekczyk) [inline features/marketing-layouts/components/MarketingTemplateBenefitBox.tsx:34]: key

File: pages/better-on-summer/test-page.tsx [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
           slidesToDisplay={2}
           overflow="visible"
-          slides={[
-            <Box sx={{ bg: 'red', width: '100%' }}>test</Box>,
-            <Box sx={{ bg: 'blue', width: '100%' }}>test</Box>,
-            <Box sx={{ bg: 'pink', width: '100%' }}>test</Box>,
-            <Box sx={{ bg: 'green', width: '100%' }}>test</Box>,
-            <Box sx={{ bg: 'yellow', width: '100%' }}>test</Box>,
-          ]}
+          slides={benefits.map((benefit) => (
```
Comment (piekczyk) [inline pages/better-on-summer/test-page.tsx:40]: key

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: features/marketing-layouts/components/MarketingTemplateBenefitBox.tsx (hunk 1 above)
Comment (piotrkonowrocki) [reply features/marketing-layouts/components/MarketingTemplateBenefitBox.tsx]: Fixed

File: pages/better-on-summer/test-page.tsx (hunk 2 above)
Comment (piotrkonowrocki) [reply pages/better-on-summer/test-page.tsx]: And fixed

**Review by piekczyk — APPROVED** [review]

**Pull request merged**